hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -410,39 +403,40 @@ public void testCreateNamespaceWithOwnership() throws Exception {
         "iceberg",
         PrincipalType.GROUP);
 
-    AssertHelpers.assertThrows(
-        String.format(
-            "Create namespace setting %s without setting %s is not allowed",
-            HiveCatalog.HMS_DB_OWNER_TYPE, HiveCatalog.HMS_DB_OWNER),
-        IllegalArgumentException.class,
-        () -> {
-          try {
-            createNamespaceAndVerifyOwnership(
-                "create_with_owner_type_alone",
-                ImmutableMap.of(HiveCatalog.HMS_DB_OWNER_TYPE, PrincipalType.USER.name()),
-                "no_post_create_expectation_due_to_exception_thrown",
-                null);
-          } catch (TException e) {
-            throw new RuntimeException("Unexpected Exception", e);
-          }
-        });
-
-    AssertHelpers.assertThrows(
-        "No enum constant " + PrincipalType.class.getCanonicalName(),
-        IllegalArgumentException.class,
-        () -> {
-          try {
-            createNamespaceAndVerifyOwnership(
-                "create_with_invalid_owner_type",
-                ImmutableMap.of(
-                    HiveCatalog.HMS_DB_OWNER, "iceberg",
-                    HiveCatalog.HMS_DB_OWNER_TYPE, "invalidOwnerType"),
-                "no_post_create_expectation_due_to_exception_thrown",
-                null);
-          } catch (TException e) {
-            throw new RuntimeException("Unexpected Exception", e);
-          }
-        });
+    assertThatThrownBy(
+            () -> {
+              try {
+                createNamespaceAndVerifyOwnership(
+                    "create_with_owner_type_alone",
+                    ImmutableMap.of(HiveCatalog.HMS_DB_OWNER_TYPE, PrincipalType.USER.name()),
+                    "no_post_create_expectation_due_to_exception_thrown",
+                    null);
+              } catch (TException e) {
```

Comment:
I think we should remove those try-catch blocks as they are not needed

Reply:
fixed, thanks!